.env.local.example

```diff
@@ -1,8 +1,13 @@
-# Rename file to .env.local and populate values
+# Rename file to .env.local (or .env) and populate values
```

Comment:
Next.js lists `.env` as a file that should be added to version control: https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables

so we should not encourage people to use it for adding secrets. (`.env.local.example` is for the dev app anyway, not for running the tests)

Reply:
The problem with this that Prisma doesn't pick up .env.local file.

Reply:
BTW The NEXTAUTH_ prefixes are deliberate in this case (and don't want to change them).

I actually want to do more with them, but is probably best to expand on another time.

Reply:
Yeah, it's a bummer: https://github.com/prisma/prisma/issues/3865 Let it be as-is for now then. Will ping the Prisma folks